frontend/packages/knative-plugin/src/components/add/ShowResourceAlert.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+interface ShowResourceAlertProps {
+  title: string;
+  resource: FirehoseResource;
+  message?: string;
+}
+
+const ShowResourceAlert: React.FC<ShowResourceAlertProps> = ({ title, message, resource }) => {
+  const [data, loaded, loadError] = useK8sWatchResource<K8sResourceKind[]>(resource);
```

Comment:
@invincibleJai this is needed once when the page is loaded. I think a get `useK8sGet` request would do just fine. WDYT?

Reply:
i thought `watch` will be good as if user creates a kn service in the same namespace, alert will go once it's there with `get` in that scenario even of service get created post load, it'll be showing alert.

let me know